mailscanner/functions.php

```diff
@@ -4682,9 +4689,8 @@ function printTenMinutesGraph()
      SELECT
       timestamp AS xaxis,
       1 as total_mail,
-      virusinfected AS total_virus,
-      isspam AS total_spam,
-      size AS total_size
+      sum(virusinfected+nameinfected+otherinfected) AS total_virus,
+      sum(isspam+ishighspam+issaspam+isrblspam+spamblacklisted) AS total_spam
```

Comment:
I don't think that this sum will do what you want: if you look at [mailscanner code](https://github.com/MailScanner/v5/blob/b02c2fbbfd7131a2b7803f72c0129b2f46c7255e/common/usr/share/MailScanner/perl/MailScanner/Message.pm) isspam is always set to 1 in any case the massage is spam (low, high rbl, etc...)